--- target/linux/x86/image/Makefile	2020-12-17 11:19:18.282908065 +0800
+++ target/linux/x86/image/Makefile	2020-12-17 11:19:11.290908331 +0800
@@ -242,6 +242,10 @@
 ifneq ($(CONFIG_TARGET_IMAGES_GZIP),)
   define Image/Build/gzip/ext4
 	$(call Image/Build/gzip,ext4)
+    ifneq ($(CONFIG_VMDK_IMAGES),)
+		$(if $(CONFIG_GRUB_IMAGES),gzip -f9n $(BIN_DIR)/$(IMG_PREFIX)-combined-ext4.vmdk)
+		$(if $(CONFIG_EFI_IMAGES),gzip -f9n $(BIN_DIR)/$(IMG_PREFIX)-combined-ext4-efi.vmdk)
+    endif
   endef
   define Image/Build/gzip/squashfs
 	$(call Image/Build/gzip,squashfs)
