--- feeds.conf.default	2020-09-03 14:28:13.062438228 +0800
+++ feeds.conf.default	2020-09-03 14:28:52.725267787 +0800
@@ -8,4 +8,4 @@
 #src-git management https://github.com/openwrt-management/packages.git
 #src-git oldpackages http://git.openwrt.org/packages.git
 #src-link custom /usr/src/openwrt/custom-feed
-#src-git helloworld https://github.com/fw876/helloworld
+src-git helloworld https://github.com/fw876/helloworld
