--- scripts/download.pl	2022-11-15 20:21:21.017292271 +0800
+++ scripts/download.pl	2022-11-15 20:22:42.626067454 +0800
@@ -107,7 +107,7 @@
 	my $filename = shift;
 
 	if ($download_tool eq "curl") {
-		return (qw(curl -f --connect-timeout 20 --retry 5 --location),
+		return (qw(curl -f --connect-timeout 20 --max-time 240 --retry 5 --location),
 			$check_certificate ? () : '--insecure',
 			shellwords($ENV{CURL_OPTIONS} || ''),
 			$url);
