--- feeds.conf.default	2023-02-21 03:34:16.467061200 +0000
+++ feeds.conf.default	2023-02-21 03:36:38.141604503 +0000
@@ -8,3 +8,5 @@
 #src-git targets https://github.com/openwrt/targets.git
 #src-git oldpackages http://git.openwrt.org/packages.git
 #src-link custom /usr/src/openwrt/custom-feed
+src-git passwallpkgs https://github.com/xiaorouji/openwrt-passwall.git;packages
+src-git passwallluci https://github.com/xiaorouji/openwrt-passwall.git;luci
